draft-ietf-tls-esni.md

```diff
@@ -476,6 +473,37 @@ contain a "cached_info" extension {{!RFC7924}} with a CachedObject
 entry whose CachedInformationType is "cert", since this indication
 would divulge the true server name.
 
+## Recommended Padding Scheme {#padding}
+
+Future extensions could reveal sensitive information through their length. Consequently, 
+padding should be flexible and support arbitrary extension changes. 
+
+For each field in the inner ClientHello, clients need to determine how much to
+pad given the semantics of that field.  For example, if a client can propose
+various ALPN values, it could add padding to round up to the longest of those.
```

Comment:
Is this part right? Do clients need to consider padding for each field, or just the entire encrypted ClientHelloInner?

Reply:
Yep -- this is what was discussed way up above in this PR.

Reply:
Ok, I will re-read this PR, since I don't get it.

But, even so, I think this section needs a rationale. It says "clients need to determine how much to pad given the semantics of that field", but there is no reason given.

Reply:
I moved that rationale down to the security considerations section as suggested earlier. It could be moved back up, but basically anything where the length could be sensitive may need padding.